tests/opentrace/test_settings.py

```diff
@@ -0,0 +1,37 @@
+import opentracing
+import pytest
+import unittest
+
+from ddtrace.settings import ConfigException
+from ddtrace.opentrace import Config
+
+
+class OpenTraceConfigTest(unittest.TestCase):
```

Comment:
Because we're leveraging `pytest` (finally!), you can just use extend `object` without using `unittest`. Here an example: https://docs.pytest.org/en/latest/getting-started.html#group-multiple-tests-in-a-class